unit_test/transforms/test_mapper.py

```diff
@@ -94,3 +94,20 @@ def test_replace_joint_in_connected_assets_error(self):
         # assert
         self.assertIsInstance(cm.exception, RuntimeError)
         self.assertEqual(cm.exception.args[0], "Error in replacing joint in connected assets.")
+
+    def test_component_with_4_connection_points(self):
+        """Method to test the to entity mapper class with 4 connection points in a asset."""
+        esdl_file_path = Path(__file__).parent / ".." / ".." / "testdata" / "simple_heat_pump.esdl"
+        esdl_file_path = str(esdl_file_path)
```

Comment:
Waarom zetten we het trouwens niet zo op dat pyesdl_from_file zowel str, als Paths kan lezen?

Reply:
ja, ik heb #161  hiervoor aangemaakt. Maar heeft mij betrefd lage prio